api/document/serializers/content.py

```diff
@@ -14,11 +13,11 @@ class NestableAnnotation:
     """Wraps an Annotation with references to annotations it contains and its
     parent."""
     def __init__(self, annotation: Annotation,
-                 parent: Optional['NestedAnnotation']):
+                 parent: Optional['NestableAnnotation']) -> None:
         self.annotation = annotation
         # NestableAnnotations point up to the parent and down to children
         self.parent = parent
-        self.children: List[NestedAnnotation] = []
```

Comment:
I have no idea how this ever even compiled since `NestedAnnotation` isn't even a symbol in this file, as far as I can tell.